examples/python-flask/README.md

```diff
@@ -0,0 +1,46 @@
+# Flask Example
+
+This project demonstrates use of verifiable credentials and presentation  for an
+application.
+
+## Dependencies
+
+- Python 3
+- Pip
+- Python 3 virtual environment
+
+```bash
+$ sudo apt update
+$ sudo apt install -y python3.6 python3-pip python3-virtualenv
+```
+
+### Python dependencies
+
+- flask-qrcode
+- Flask
+- didkit
+
+```bash
+$ python3 -m pip install flask-qrcode flask
+```
+
+### Building DIDKit
+
+DIDKit is used to handle credentials and presentations, since it's not yet
+publically available in PyPI manual installation is required.
+
+To do so got to the `lib/` folder inside this repository and run:
+```bash
+$ make -C lib ../target/test/python.stamp
+```
+
+## Running
+
+For the first time running you will need to run the migrations,
+this can be accomplished by running the following command:
+
+To start the server just run:
+
+```bash
+$ FLASK_APP=didkit_flask.py python3 didkit_flask.py
```

Comment:
Unfortunately, ran and got this. Perhaps we can add this as a dependency that pip install? 

```
 % cd examples/python-flask
~/didkit/examples/python-flask
 % FLASK_APP=didkit_flask.py python3 didkit_flask.py
Traceback (most recent call last):
  File "didkit_flask.py", line 3, in <module>
    from issue_credential import issueCredential
  File "/home/test/didkit/examples/python-flask/issue_credential.py", line 1, in <module>
    from django.core.files import File
ModuleNotFoundError: No module named 'django'
```

Furthermore, we should instruct the user to `cd examples/python-flask`.

Reply:
Is it possible to make the Flask example listen on 127.0.0.1 instead of 0.0.0.0? Could be better from a security standpoint.

Reply:
This was my mistake, I forgot to replace this dependency, already fixed.